packages/amplify-cli-core/src/permissionsBoundaryState.ts

```diff
@@ -1,14 +1,20 @@
-import { stateManager } from './state-manager';
 import _ from 'lodash';
-import { $TSObject } from '.';
+// eslint-disable-next-line import/no-cycle
+import { stateManager } from './state-manager';
+// eslint-disable-next-line import/no-cycle
+import { $TSAny, $TSObject } from '.';
 
-let preInitTeamProviderInfo: any;
+let preInitTeamProviderInfo: $TSAny;
```

Comment:
nit: I think it's ideal to use `$TSTeamProviderInfo`, but it's just an alias for `$TSAny` currently.

Reply:
with any luck, team provider info won't exist in a few months ;) but I'll update this for now